cmd/gateway-sia-cachelayer.go

```diff
@@ -0,0 +1,685 @@
+/*
+ * (C) 2017 David Gore <dvstate@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/boltdb/bolt"
```

Comment:
Move the `github.com` imports as mentioned here - you can also use `goimports` to do this for you. https://github.com/golang/go/wiki/CodeReviewComments#imports